imod/viewers/viewers.py

```diff
@@ -122,6 +122,10 @@ def __init__(self, obj, protocol=None, **kwargs):
 
             cmd += f"-a {angleFilePath} -m {outputTSPath} {fidFileName}"
 
+        # A path called from the object browser
+        elif isinstance(obj, str):
```

Comment:
aah, this is what this case is for! nice!

Reply:
Yes, right click on the File browser (not object browser) will send the path to the viewer